sdk/spring/spring-integration-azure-core/src/main/java/com/azure/spring/integration/core/handler/DefaultMessageHandler.java

```diff
@@ -248,7 +248,7 @@ private Map<String, Object> buildPropertiesMap() {
      *
      * @param callback the call back
      */
-    public void setSendCallback(ListenableFutureCallback<Void> callback) {
+    public void setSendCallback(BiConsumer<Void, Throwable> callback) {
```

Comment:
This is a public API change right?

Reply:
Yes, will bring breaking changes.

Reply:
Can we refer to other projects?

Reply:
The Spring project itself still uses this callback interface, shall we also keep using this first?
Below are the referred links:
SF 6: https://github.com/spring-projects/spring-framework/blob/main/spring-webmvc/src/main/java/org/springframework/web/servlet/mvc/method/annotation/DeferredResultMethodReturnValueHandler.java#L79-L108
SF 5.3.23 https://github.com/spring-projects/spring-framework/blob/v5.3.23/spring-webmvc/src/main/java/org/springframework/web/servlet/mvc/method/annotation/DeferredResultMethodReturnValueHandler.java#L78-L107

Spring Kafka 3.0.0-M6 https://github.com/spring-projects/spring-kafka/blob/v3.0.0-M6/spring-kafka/src/test/java/org/springframework/kafka/core/KafkaTemplateTests.java#L397-L415
Spring Kafka 2.8.9 https://github.com/spring-projects/spring-kafka/blob/v2.8.9/spring-kafka/src/test/java/org/springframework/kafka/core/KafkaTemplateTests.java#L389-L414